### ISSUE TITLE
Additional multi-dc test to see if multi-dc-heartbeat takes over during unreachable or split

### ISSUE THREAD

**Issue by ktoso** [akka/akka#23345]
Similar test to https://github.com/akka/akka/pull/23342 however with the added difficulty of happening during a network split or unreachability across DCs. Need to also figure out and document exact semantics we have there while we're at it.

**Comment by agolubev** [comment]
@ktoso, should this be closed?

**Comment by ktoso** [comment]
No, as the comment explains this is about a similar to the existing test AFAIR
